gen_import_utils.py

```diff
@@ -47,6 +47,27 @@ def replace_slashes_in_dict(json):
         elif isinstance(value, dict):
             replace_slashes_in_dict(value)
 
```

Comment:
used in picturae_importer to truncate paths in the jpg_path column to the last n elements/folders.